index.ts

```diff
@@ -258,6 +256,56 @@ export class Project {
       dep.baseDir = path.join(this.baseDir, 'node_modules', dep.name);
       dep.writeSync();
     }
+    for (let [name, { dir: target }] of this.dependencyLinks) {
+      this.writeLinkedPackage(name, target);
+    }
+  }
+
+  private writeLinkedPackage(name: string, target: string) {
+    let targetPkg = require(path.join(target, 'package.json'));
+    let peers = new Set(Object.keys(targetPkg.peerDependencies ?? {}));
+    let destination = path.join(this.baseDir, 'node_modules', name);
+
+    if (peers.size === 0) {
```

Comment:
What happens when a given linked dependency itself has no peer-dependencies, but one of it's dependencies does?

Scenario:

```
a -> e
a -> b -> c -[peer]-> e
```

Reply:
Replied to this via email and broke the threading, see https://github.com/stefanpenner/node-fixturify-project/pull/50#issuecomment-918513614